packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -21,6 +22,7 @@ const UTM_PREFIX = 'fx-';
 
 const X_SES_CONFIGURATION_SET = 'X-SES-CONFIGURATION-SET';
 const X_SES_MESSAGE_TAGS = 'X-SES-MESSAGE-TAGS';
+var isNewEmails = false;
```

Comment:
Since feature flagging is not in place yet, I'm using this temporary variable (for development purpose) to check if the template is new or not